packages/developer-portal/src/components/pages/webhooks/webhooks.tsx

```diff
@@ -204,7 +204,7 @@ export const DeveloperWebhooks = () => {
         <FormSubHeading>
           This system is designed to flexibly work with how your application is built and deployed. If you wish, you can
           set up a single endpoint to catch all topics for all customers. Alternatively, you may wish to set up a
-          different webhook subscription per topic or per customer. For more information about Webhooks, please see our
+          different webhook subscription per topic or per customer. For more information about Webhooks, please see our{' '}
```

Comment:
pls remove `{' '}`

Reply:
Hi @tanphamhaiduong, `{' '} is placed intentionally to add spacing between webhooks link and paragraph

Reply:
@phmngocnghia should we change to `&nbsp;` ?